x-pack/plugins/observability_solution/entity_manager/server/lib/entities/helpers/generate_component_id.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EntityDefinition } from '@kbn/entities-schema';
+import {
+  ENTITY_BASE_PREFIX,
+  ENTITY_LATEST,
+  ENTITY_HISTORY,
+  ENTITY_SCHEMA_VERSION_V1,
+} from '../../../../common/constants_entities';
+
```

Comment:
I'm wondering if we should be passing in the schema version into the helpers that generate these kind of names?

Reply:
i think it's fine in the constant like this